### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by AbigiyaTY**
# Responsive page

In this milestone I:

- Add layers in index.html that are used to make the page responsive.
- Add media queries and styling to the CSS to make the page responsive.
- Fixed the alignment of the page.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -309,160 +413,606 @@ a {
...
+  }
+
+  .my-email {
+    margin-top: 22px;
+    margin-left: 0;
+  }
+}
+
+@media only screen and (min-width: 1250px) {
+  body {
```
Comment (tufoinnkuo10) [inline css/style.css]: - [ ] kindly remove this as it make your portfolio not to display on screen bigger than 1250px

File: index.html [hunk 2]
```diff
@@ -195,35 +216,42 @@ <h2>CONTACT US</h2>
                 <br />
                 <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required /><br />
                 <br />
-                <textarea name="comments" id="comments" placeholder="Enter text here" maxlength="500" rows="8" cols="40" required></textarea>
-                <input class="button2" type="submit" value="See Project" />
+                <textarea name="comments" id="comments" placeholder="Enter text here" maxlength="500" rows="8" cols="46" required></textarea>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] maybe you didn't notice. The mobile version is supposed to still look ok. kindly align the input fields
Screenshot (214)_LI

File: index.html [hunk 3]
```diff
@@ -20,14 +20,20 @@
...
-            </a>
+            <h3 class="name1"><a href="#">AB</a></h3>
             <i class="hamburger-menu fa fa-bars"></i>
+            <ul class="hamburger-list">
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+                <li>
+                    <a href="#"><i class="fa fa-envelope-o"></i> </a>
+                </li>
```
Comment (tufoinnkuo10) [inline index.html:31]: - [ ] please make the headline section responsive. When switching to the desktop version, the text goes one-sided. Kindly fix and centralize it. 
- [ ] also make your heading bolder, so they look more like the figma version
your version 
Screenshot (217)
figma
Screenshot (215)

File: index.html [hunk 4]
```diff
@@ -20,14 +20,20 @@
...
+            </ul>
         </div>
     </header>
     <section class="header-container-top">
```
Comment (tufoinnkuo10) [inline index.html:35]: - [ ] you are missing the work indicator and social icons in your desktop version. kindly fix that
your version
Screenshot (218)_LI
figma version
Screenshot (219)_LI

File: index.html [hunk 5]
```diff
@@ -47,142 +53,157 @@ <h2>Software Developer</h2>
...
+                        </ul>
+                    </div>
+                    <div class="ellipse2"></div>
+                </div>
+            </div>
+            <div class="ellipseDS"></div>
+        </div>
+    </section>
+    <section class="work">
+        <h2 class="work-title">MY RECENT WORK</h2>
```
Comment (tufoinnkuo10) [inline index.html:138]: - [ ] please make your work section cards bigger and well centralized, so it looks more like the figma version

File: index.html [hunk 6]
```diff
@@ -47,142 +53,157 @@ <h2>Software Developer</h2>
...
             </div>
+            <button class="button1" type="button">See Project</button>
         </div>
     </section>
     <section>
         <div class="Contact-Us">
             <div class="contact-title">
-                <h2>CONTACT US</h2>
+                <h2 class="contact-title-sm">CONTACT US</h2>
+                <h2 class="contact-title-LG">Let’s chat!</h2>
```
Comment (tufoinnkuo10) [inline index.html:246]: - [ ] your contact section and footer looks really small. Kindly follow the font size and font weight of the figma desktop version

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @AbigiyaTY 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ figma design followed
✅responsive

## Required Changes ♻️♻️
Check the comments under the review
- [ required] please add a description of what your project is all about in your readme

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 7]
```diff
@@ -195,35 +254,42 @@ <h2>CONTACT US</h2>
                 <br />
                 <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required /><br />
                 <br />
```
Comment (Ismailco) [inline index.html:258]: - [ ] As the previous reviewer mentioned, you need to fix this design flow, kindly make sure the name and email inputs are aligned properly so your form is designed properly 

Screenshot_20220708_042900

File: index.html [hunk 8]
```diff
@@ -20,14 +20,20 @@
...
+                </li>
+            </ul>
         </div>
     </header>
     <section class="header-container-top">
-        <div class="ellipse"></div>
+        <div class="lgImg1"><img src="./images/lg-img1.svg" alt="#" /></div>
         <div class="header-container">
             <div class="content">
                 <div class="personal-description">
```
Comment (Ismailco) [inline index.html:39]: - [ ] This section should be centered in the desktop view it's aligned more to the left, As the previous reviewer suggests kindly try to center the text on the headline section and it's preferable to add some more font size to the `h1` so your design looks like the Figma one

Screenshot_20220708_043913

File: index.html [hunk 9]
```diff
@@ -195,35 +254,42 @@ <h2>CONTACT US</h2>
                 <br />
                 <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required /><br />
```
Comment (Ismailco) [inline index.html:257]: - [ ] [OPTIONAL] In the desktop version the contact form is not centered on the page kindly make sure it does, so your design looks more semantic and follows the Figma style

Screenshot_20220708_044300

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @AbigiyaTY, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (AbigiyaTY) [reply index.html:256]: Hey @Ismailco 
 
Thanks for your review.

My question is that I have fixed this error and it works perfectly in my advice, would you mind checking once again and tell me which screen size works perfectly in all screen sizes on my side thanks.

File: index.html (hunk 8 above)
Comment (AbigiyaTY) [reply index.html:39]: Hey @Ismailco

I have also fixed this error it works perfectly in my advice, would you mind checking once again and tell me which screen size works perfectly in all screen sizes on my side thanks.

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @AbigiyaTY  👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**